docs/src/fonts/index.css

```diff
@@ -44,58 +44,76 @@
   font-family: 'SF Mono';
   font-weight: normal;
   src: local('SFMono-Regular'), local('ui-monospace');
-  /* Match 13px Unica to 12px SF Mono */
-  size-adjust: calc(100% * 12 / 13);
+  /* Match 14px Unica to 13px SF Mono */
+  size-adjust: calc(100% * 13 / 14);
 }
 
 @font-face {
   font-family: 'SF Mono';
   font-weight: bold;
   src: local('SFMono-Bold'), local('ui-monospace');
-  /* Match 13px Unica to 12px SF Mono */
-  size-adjust: calc(100% * 12 / 13);
+  /* Match 14px Unica to 13px SF Mono */
+  size-adjust: calc(100% * 13 / 14);
 }
 
 @font-face {
   font-family: 'Menlo';
   font-weight: normal;
   src: local('Menlo-Regular');
-  /* Match 13px Unica to 11.5px Menlo */
-  size-adjust: calc(100% * 11.5 / 13);
+  /* Match other monospace fonts to SF Mono in width */
+  size-adjust: calc(100% * 13.25 / 14);
```

Comment:
@colmtuite I realised I was looking at a much wider SF Mono while you were looking at a much narrower Menlo 

Bumped it 7% larger, does it feel OK now or do you want to go larger?

Reply:
@vladmoroz I'd go a tiny bit larger still. Maybe 13.5px or something like that.